src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final int radius;
+    private final int index = 2;
```

Comment:
move this to line 14, don't create a field, and naming is unsuitable, it's not an index